google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -1255,6 +1260,447 @@ public static SignUrlOption withQueryParams(Map<String, String> queryParams) {
     }
   }
 
+  class V4PostPolicyOption implements Serializable {
+    private static final long serialVersionUID = 8150867146534084543L;
+    private final V4PostPolicyOption.Option option;
+    private final Object value;
+
+    enum Option {
+      PATH_STYLE,
+      VIRTUAL_HOSTED_STYLE,
+      BUCKET_BOUND_HOST_NAME,
+      SERVICE_ACCOUNT_CRED
+    }
+
+    private V4PostPolicyOption(Option option, Object value) {
+      this.option = option;
+      this.value = value;
+    }
+
+    V4PostPolicyOption.Option getOption() {
+      return option;
+    }
+
+    Object getValue() {
+      return value;
+    }
+
+    /**
+     * Provides a service account signer to sign the policy. If not provided an attempt will be made
+     * to get it from the environment.
+     *
+     * @see <a href="https://cloud.google.com/storage/docs/authentication#service_accounts">Service
+     *     Accounts</a>
+     */
+    public static V4PostPolicyOption signWith(ServiceAccountSigner signer) {
+      return new V4PostPolicyOption(V4PostPolicyOption.Option.SERVICE_ACCOUNT_CRED, signer);
+    }
+
+    /**
+     * Use a virtual hosted-style hostname, which adds the bucket into the host portion of the URI
+     * rather than the path, e.g. 'https://mybucket.storage.googleapis.com/...'. The bucket name
+     * will be obtained from the resource passed in.
```

Comment:
will be --> is
per Google Tech Writing Guidelines
https://developers.google.com/style/tense